### ISSUE TITLE
Appium cannot find Infragistics WPF controls

### ISSUE THREAD

**Issue by DanielBMann9000** [Microsoft/WinAppDriver#254]
Similar to problems with other UI test automation frameworks (Coded UI, etc). Searching by name or accessibility ID for items contained within Infragistics controls (such as their grid view control) does not work. Items within the control appear to not be discoverable by Appium -- the entire control is "opaque".  Inspect sees the items within the control in UIA mode, but not in MSAA mode.

Tried Appium Desktop Inspector -- controls do not show up. The entire Infragistics control is just one big "pane" element with no children.

Is there any workaround?

**Comment by timotiusmargo** [comment]
Hi @DanielBMann9000,

WinAppDriver is suppose to see everything that [inspect.exe](https://github.com/Microsoft/WinAppDriver/blob/v1.0-RC/README.md#inspecting-ui-elements) tool see in **UIA mode**. If this is not what you observe, could you please provide a simple app that we can repro the issue with?

**Comment by DanielBMann9000** [comment]
@timotiusmargo Unfortunately, I observed the behavior with a client's application. I do not have access to the application, nor do I have a license to Infragistics controls. I will reach out to them and see if they can provide a MCVE for you.

**Comment by hassanuz** [comment]
Closing issue due to time gap.
